src/checkers/inference/solver/backend/encoder/AbstractConstraintEncoder.java

```diff
@@ -0,0 +1,32 @@
+package checkers.inference.solver.backend.encoder;
+
+import checkers.inference.solver.frontend.Lattice;
+import checkers.inference.util.ConstraintVerifier;
+
+/**
+ * Created by mier on 07/11/17.
+ */
+public abstract class AbstractConstraintEncoder<ConstraintEncodingT> {
+
+    protected final Lattice lattice;
+    protected final ConstraintVerifier verifier;
+    protected final ConstraintEncodingT emptyValue;
+    // Clauses that are always false. Used to model unsatisfactory constraint between two
```

Comment:
This code comment seems related to MaxSat encoding and is not general for AbstractConstraintEncoder.

Reply:
Done.